accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx

```diff
@@ -957,6 +962,94 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
+                      </p>
+                    </li>
+                    <li>
+                      <p style={{ marginTop: "0rem" }}>
+                        Press the <strong>'tab'</strong> key to navigate to the <strong>issue grouping</strong> and use the <strong>'enter'</strong> to open or close the issue grouping. Press the <strong>'tab'</strong> key to navigate to each issue, and press the <strong>'enter'</strong> key to select any issue.
+                      </p>
+                    </li>
+                    <li>
+                      <p style={{ marginTop: "0rem" }}>
+                        Press the <strong>'tab'</strong> key to move to the <strong>Learn more</strong> link or to move to the next issue.
+                      </p>
+                    </li>
+                  </ul>  
+                <li>
+                  Use the <strong>'headings'</strong> hierarchy or the implemented <strong>'landmarks'</strong> to quickly navigate from one section to another. The list of implemented landmarks are as follows:
+                </li>
+                  <ul
+                    style={{
+                      listStyleType: "circle",
+                      marginInlineStart: "2rem",
+                    }}
+                  >
+                    <li>
+                      <p style={{ marginTop: "0rem" }}>
+                        The <strong>IBM Accessibility Assessment</strong> or the <strong>IBM Accessibility Checker</strong>, main landmark: contains the main functionality of the tool in each view and includes,
+                      </p>
+                      <ul
+                          style={{
+                            listStyleType: "disc",
+                            marginInlineStart: "2rem",
+                          }}
+                        >
+                          <li>
+                            <p style={{ marginTop: "0rem" }}>
+                              The <strong>Issue Count</strong>, region: contains the issue count by issue type as well as the total number of issues found.
+                            </p>
+                          </li>
+                          <li>
+                            <p style={{ marginTop: "0rem" }}>
+                              The <strong>Issue List</strong>, region: contains the list of issue grouped by Element Roles, Checkpoints or by Rules.
+                            </p>
+                          </li>
+                        </ul>
+                    </li>
+                    <li>
+                      <p style={{ marginTop: "0rem" }}>
+                        The <strong>Scan Summary</strong> or the <strong>Issue help</strong>, aside or the complementary landmark: in the Accessibility Assessment view, contains the scan summary, after the scan completes or shows the issue help when any issue is selected.
```

Comment:
I'm finding this unclear when written as one item.  Will users perceive these as a single aside or as two different ones?  If different, perhaps separate into two items.

```suggestion
                        The Accessibility Assessment view <strong>Scan Summary</strong> aside: contains the scan summary, after the scan completes.
                        
             The Accessibility Assessment view <strong>Issue Help</strong> aside: contains the issue help when any issue is selected.
```